client/extensions/woocommerce/app/products/product-form-variations-modal.js

```diff
@@ -94,6 +109,32 @@ class ProductFormVariationsModal extends React.Component {
 					</FormFieldSet>
 
 					<FormLabel>
+						{ translate( 'Dimensions & weight' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **77** times:
`translate( 'Dimensions' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/woocommerce-move-variation-dimensions)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).